htdocs/admin/system/dolibarr.php

```diff
@@ -459,7 +460,12 @@
 
 		print '<tr class="oddeven">';
 		print '<td class="tdoverflowmax300">'.$obj->name.'</td>'."\n";
-		print '<td class="tdoverflowmax300">'.dol_escape_htmltag($obj->value).'</td>'."\n";
+		if (strpos(strtolower($obj->name), 'password') || strpos(strtolower($obj->name), 'token') || strpos(strtolower($obj->name), 'secret') || strpos(strtolower($obj->name), '_pw')) {
+			$value = preg_replace('/./i', '*', $obj->value);
```

Comment:
If we introduce a way to hide value on screen (good), we must also introduce a button/picto "reveal password". Because this page is a page for admin users and they used it to get the information...